lib/Command/CheckUser.php

```diff
@@ -82,7 +78,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 			$uid = $input->getArgument('ocName');
 			$this->isAllowed($input->getOption('force'));
 			$this->confirmUserIsMapped($uid);
-			$exists = $this->backend->userExistsOnLDAP($uid);
+			$exists = $this->backend->userExists($uid);
```

Comment:
The implementation now always talks to LDAP but caches the result per request.

Reply:
Actually userExistsOnLdap has been moved to User_LDAP. userExists tries to determine the answer based on the cache in the LDAP UserManager. It will fall back to LDAP just as before.

Reply:
Ok, so the redis cache has been removed and we cache the information per request... fair enough 👍